contents/handbook/strategy/strategy.md

```diff
@@ -4,53 +4,165 @@ sidebar: Handbook
 showTitle: true
 ---
 
-> PostHog's mission is to increase the number of successful products in the world.
+## TL;DR
+Our mission is to **_“Increase the number of successful products in the world”_**. Our first step on their journey is to prove product market fit with 5 reference customers. Today we're closest to PMF with products which need to self-host and are in the growth phase of their lifecycle; they need us to absolutely "Nail Funnels" in order to make their products successful.
```

Comment:
For the purpose of external positioning, I'd reference the fact that thousands of teams use us already, we already have tons of paying customers

Reply:
I think we also need to include something that tells the story given this doc hasn't got any context and it could drag investors / competitors to a bad spot. This is _a bit fluffier_ but the tradeoff is probably worth it. How about:

Our mission is to **_“Increase the number of successful products in the world”_**.

We started by building an open-source product analytics platform with all the basic things you'd need to understand user behavior - funnels, trends, session recordings, feature flags and many more features. Within a year we had thousands of customers using us and we started generating revenue.

Our next step on our journey is to prove a deeper level of product-market fit with 5 reference customers. Today we're closest to product market fit with products which need to self-host and are in the growth phase of their lifecycle; they need us to absolutely "Nail Funnels" in order to make their products successful.